doc/build-unix.md

```diff
@@ -69,7 +69,7 @@ Build requirements:
 
 Options when installing required Boost library files:
 
-1. On at least Ubuntu 14.04+ and Debian 7+ there are generic names for the
+1. On at least Ubuntu 14.04+ ( > kernel 3.13 )  and Debian 7+( > kernel v3.2.41)  there are generic names for the
```

Comment:
kernel versions determine what are each release.
this can help those who modify the kernel to kernel.org releases and not modifications of debian or canonical.